src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -95,18 +130,31 @@ export class CalciteDropdown {
     }
   }
 
-  componentWillLoad() {
-    // get initially selected items
-    this.updateSelectedItems();
+  disconnectedCallback(): void {
+    this.destroyPopper();
   }
 
-  render() {
+  render(): VNode {
     const { maxScrollerHeight } = this;
     const dir = getElementDir(this.el);
     return (
-      <Host dir={dir} tabIndex={this.disabled ? -1 : null}>
-        <slot name="dropdown-trigger" aria-haspopup="true" aria-expanded={this.active.toString()} />
+      <Host dir={dir}>
         <div
+          class="calcite-dropdown-trigger-container"
+          ref={(el) => (this.referenceEl = el)}
+          tabIndex={this.disabled ? -1 : null}
```

Comment:
Moving this to the container instead of `:host` allows you to tab through to a disabled component - could we move that back (or set on both `:host` and this element)?